build/setup.d/20-install-packages.sh

```diff
@@ -6,7 +6,7 @@
 pkgs="
 auditd
 build-essential
-docker-engine=1.11.1-0~trusty
+docker-engine=1.12.6-0~ubuntu-trusty
```

Comment:
Docker is a beast and has to be proven working in all cases for us. We hit with every version change a lot of problems with docker, so if there is no reason then I would prefer no upgrade. I am ok, if there is a reason to upgrade that you upgrade it and it seems that you have a valid reason.